### ISSUE TITLE
Further information is requested

### ISSUE THREAD

**Issue by oula-le** [junyuchen245/Semi-supervised_FCM_Loss_for_Segmentation#1]
Hello, can you put the code of network framework structure on it?Can you give me the training set and the test set?

**Comment by junyuchen245** [comment]
Hi @oula-le 

The loss function is independent of network architecture. You can use it with the conventional U-Net, but keep in mind that it requires the softmax output from the network.

Unfortunately, I cannot distribute the dataset, as the data was acquired as part of an IRB protocol and is not approved for public release. 

Thanks,
Junyu
